src/AudioUnit/AudioUnit.cs

```diff
@@ -1106,13 +1106,8 @@ public AudioUnitStatus SetScheduledFiles (AudioFile[] audioFiles)
 			for (int i = 0; i < count; i++)
 				handles [i] = audioFiles [i].Handle;
 
-			GCHandle gch = GCHandle.Alloc (handles, GCHandleType.Pinned);
-			IntPtr ptr = gch.AddrOfPinnedObject ();
-
-			var status = AudioUnitSetProperty (Handle, AudioUnitPropertyIDType.ScheduledFileIDs, AudioUnitScopeType.Global, 0, ptr,  Marshal.SizeOf (ptr) * count);
-
-			gch.Free ();
-			return status;
+			fixed (IntPtr* ptr = &handles [0])
```

Comment:
I can't take the address of an array:

    test.cs(7,25): error CS0208: Cannot take the address of, get the size of, or declare a pointer to a managed type `byte[]'

for this code:

    var bytes = new byte [0];
    fixed (byte* ptr = &bytes) {}